pkg/config/defaults.go

```diff
@@ -58,7 +58,7 @@ var DefaultNodeConfig = Config{
 		Seeds:         "",
 	},
 	Node: NodeConfig{
-		Aggregator:        true,
+		Aggregator:        false,
```

Comment:
any reason for changing this? by default the start command runs as aggregator

Reply:
i moved it to false so new nodes spinning up dont need to be explicit. this is because how the file signer works it requires a pass phrase when generating the key, but for 99% of nodes this isnt needed